test-packages/support/audit-assertions.ts

```diff
@@ -38,15 +41,31 @@ export function setupAuditTest(hooks: NestedHooks, opts: () => AuditBuildOptions
       pathRewriter = p => getRewrittenLocation(o.app, p);
     }
     expectAudit = new ExpectAuditResults(result, assert, pathRewriter);
+  }
+
+  hooks.before(async () => {
+    await visit();
+  });
+
+  hooks.beforeEach(assert => {
+    installAuditAssertions(assert);
+    prepareResult(assert);
   });
 
   return {
+    async rerun() {
```

Comment:
for tests that change files while vite dev is running we need to be able to rerun the vistors